docs/development.md

```diff
@@ -1,5 +1,18 @@
 # Getting started with REMS development
 
+## Installed Software
+
+In order to get started with REMS, you need to have the following software installed:
+
+   - Docker
+   - Java. Currently REMS should be working with Java versions 11 to 16
+   - leiningen. As of 14/04/2021 the project should work with lein version Leiningen 2.9.5 on Java 16 OpenJDK 64-Bit Server VM.
```

Comment:
do we need to mention the java version here again? also, please consider a timestamp format 2021-04-14, which we use elsewhere (e.g. releases)